Index: src/main/java/Basics.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/Basics.java b/src/main/java/Basics.java
--- a/src/main/java/Basics.java	
+++ b/src/main/java/Basics.java	(date 1725832784677)
@@ -180,6 +180,9 @@
          *                You can index into arrays as we do in Python
          *                (e.g. arr[i] gives you the item at index i).
          */
+        for (int i = 1; i < arr.length; i += 2) {
+            current_sum += arr[i];
+        }
 
         return current_sum;
     }
